_cards/features.md

```diff
@@ -11,7 +11,9 @@ urls:
   getstarted: "#Onboarding"  
 ---
 <div class="row">
-  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-xs-12">Using common services will jump start the development efforts of your team and lower the overall maintenance cost of your product. You can focus on building features that matter to your users. Get started today!</div>
+  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-xs-12">
```

Comment:
I can't quite remember bootstrap, but I thik you might be able to simplify this to just
```
<div class="col-lg-6 col-xs-12">
```
As the default is to look at the smallest box size and apply that upwards until a different rule applies.